internal/app/connectconformance/test_case_library.go

```diff
@@ -441,6 +453,7 @@ func populateExpectedResponse(testCase *conformancev1.TestCase) error {
 	// message in this situation, where the response data value is some fixed string (such as "no response definition")
 	// and whose request info will still be present, but we expect it to indicate zero request messages.
 	if len(testCase.Request.RequestMessages) == 0 {
+		testCase.ExpectedResponse = &conformancev1.ClientResponseResult{}
```

Comment:
This was causing a panic if you omitted `requestMessages` from a test case. Since `ExpectedResponse` was nil here, once we returned, the `assert` function in results.go was blowing up on line ~170.